packages/gatsby-source-kontent/src/sourceNodes.items.ts

```diff
@@ -54,7 +54,7 @@ const getKontentItemLanguageVariantArtifact = (
   includeRawContent: boolean,
 ): KontentItem => {
   const nodeIdString = getKontentItemNodeStringForId(
-    kontentItem.system.codename,
+    kontentItem.system.id,

```

Comment:
Could this cause some undocumented breaking change?

Reply:
Since it is beta - the breaking change among patch versions are fine. 
The upgrade will definitely require a new full build of the site.

But I could add it to the Upgrade guide 👍

Reply:
Fixed: 883b00e8b4bc0f1884d888ded8a4077d730bc8b4

Reply:
yeah, I thought about upgrade instructions, ty